src/Jh/DataImportMagento/Writer/ShipmentWriter.php

```diff
@@ -64,6 +70,17 @@ public function writeItem(array $item)
                 ->addObject($shipment->getOrder())
                 ->save();
 
+            if (isset($item['tracking_number'], $item['carrier'])) {
```

Comment:
Yes but this library is a generic library. Not your projects import. You need to consider what will happen if another project needs multiple tracks. I'm not saying you should implement it, but you should consider it. Would it be a BC to implement it in the future? How hard would it be? Surely in this case it is just a simple loop? Not something that would be hard or long to implement?